atlasdb-commons/src/main/java/com/palantir/common/visitor/Visitor.java

```diff
@@ -13,14 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.palantir.util;
-
+package com.palantir.common.visitor;
```

Comment:
the problem is actually worse than that.. internally we had a duplicate of this class with the package "com.palantir.common.visitor".  Changing the name of this one made fixing all referencing a little easier (and the name already makes a little more sense).  I've already done that fix internally (delete the duplicate and fixed all users of both).